app/src/main/java/com/spascoding/taskycourse/feature_register_screen/presentation/components/CustomOutlinedTextField.kt

```diff
@@ -0,0 +1,80 @@
+package com.spascoding.taskycourse.feature_register_screen.presentation.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import com.spascoding.taskycourse.core.constants.Colors
+import com.spascoding.taskycourse.core.constants.FontSize
+import com.spascoding.taskycourse.core.constants.RoundCorner
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun CustomOutlinedTextField(
```

Comment:
I'd never name Composables starting with `Custom`, since that doesn't tell the caller anything about its purpose and in how far it's custom. In the end, every Composable is custom, isn't it? :D 
If you're creating a Composable that represents a default styled component used across the project like here, I like to communicate that with the name instead. For example, `TaskyTextField` or `DefaultTextField`. That at least communicates that it's a component with the default style from the mockups which should be used for all components of this type across the project.